pilot/cmd/pilot-agent/main.go

```diff
@@ -808,8 +806,6 @@ func init() {
 	// See https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-component-log-level
 	proxyCmd.PersistentFlags().StringVar(&proxyComponentLogLevel, "proxyComponentLogLevel", "misc:error",
 		"The component log level used to start the Envoy proxy")
-	proxyCmd.PersistentFlags().StringVar(&dnsRefreshRate, "dnsRefreshRate", "300s",
```

Comment:
Would suggest first deprecate it rather than removing directly

Reply:
why? we support unknown flags now, so I am not sure what the use case is for setting this flag. ServiceEntry has been behaving this way for 2 releases now

Reply:
Oh, i did not know that. Then it should work.

Reply:
Yeah we added it a while back for similar reason